16_Bysheva/yadisk_automation/Program.cs

```diff
@@ -58,7 +58,10 @@ static int Main(string[] args)
                 YadiskActions.Login(browserWindow, LOGIN, Password);
 
                 //ChooseFile(browserWindow, filePath);
-                YadiskActions.DragFileFromExplorer(filePath, ElementFinder.TryFindElement(browserWindow, "последние файлы", "text", YadiskActions.DefaultTimeout).Current.BoundingRectangle.TopLeft);
+                //YadiskActions.DragFileFromExplorer(filePath, ElementFinder.TryFindElement(browserWindow, "последние файлы", "text", YadiskActions.DefaultTimeout).Current.BoundingRectangle.TopLeft);
+                var point = browserWindow.Current.BoundingRectangle.TopLeft;
+                point.Offset(100, 100);
```

Comment:
Still resolution-dependent. Do not use magic number, use appropriate control rectangle mid point.